History.md

```diff
@@ -1,4 +1,28 @@
-## 5.6.5 / 2022-07-dd
+## 5.7.0 / 2022-08-XX
+
+* Features
+  * Puma::ControlCLI - allow refork command to be sent as a request ([#2868], [#2866])
```

Comment:
This is listed under 5.6.5 too, but that looks incorrect (GitHub doesn't show that https://github.com/puma/puma/commit/6543f0959bd495b8b0326777034d6f13651dd4b3 has the v5.6.5 tag)

Will you fix that in this PR too? (I see you adjusted the date for 5.6.5)

Reply:
>but that looks incorrect (GitHub doesn't show that https://github.com/puma/puma/commit/6543f0959bd495b8b0326777034d6f13651dd4b3 has the v5.6.5 tag)

Can't seem to find that tag on any commits so I'm not sure what's right :/

Reply:
cherry-pick?  See https://github.com/puma/puma/commit/6454710f1c

Reply:
That commit also doesn't show the tag (here on GitHub) Yes it is in 5-6-stable branch though 

I'm confused as how we make releases :)